shell/src/main/java/org/apache/accumulo/shell/commands/InsertCommand.java

```diff
@@ -171,6 +171,8 @@ public Options getOptions() {
         "durability to use for insert, should be one of \"none\" \"log\" \"flush\" or \"sync\"");
     o.addOption(durabilityOption);
 
+    o.addOption(OptUtil.tableOpt("table in which data is to be inserted"));
```

Comment:
Minor grammatical nit:
```suggestion
    o.addOption(OptUtil.tableOpt("table into which data will be inserted"));
```

Reply:
No problem. Didn't quite sound right to me, but I was blanking on how to re-word it.